include/arizona_assert.hrl

```diff
@@ -0,0 +1,9 @@
+-include_lib("stdlib/include/assert.hrl").
```

Comment:
We move this here for two reasons:

1. it's a pre-condition to the internals
2. it eases consumption of this `include`